Tests/SonarQube.Bootstrapper.Tests/BootstrapperSettingsTests.cs

```diff
@@ -70,15 +43,14 @@ public void BootSettings_DownloadDirFromEnvVars()
                 AssertExpectedDownloadDir(@"tfs build\.sonarqube\bin", settings);
             }
 
-            // 3. SQ variable takes precedence over the other variables
+            // 3. CWD has least precedence over env variables
             using (EnvironmentVariableScope scope = new EnvironmentVariableScope())
             {
-                scope.SetVariable(BootstrapperSettings.SQAnalysisRootPath, "sq build");
-                scope.SetVariable(BootstrapperSettings.BuildDirectory_Legacy, "legacy tf build");
-                scope.SetVariable(BootstrapperSettings.BuildDirectory_TFS2015, "tfs build");
+                scope.SetVariable(BootstrapperSettings.BuildDirectory_Legacy, null);
+                scope.SetVariable(BootstrapperSettings.BuildDirectory_TFS2015, null);
 
                 IBootstrapperSettings settings = new BootstrapperSettings(logger);
-                AssertExpectedDownloadDir(@"sq build\.sonarqube\bin", settings);
+                AssertExpectedDownloadDir(Path.Combine(Directory.GetCurrentDirectory(), @".sonarqube\bin"), settings);
```

Comment:
P3: make ".sonarqube" and "bin" constants public (they are private at the moment) and use them in the test